code_examples/sdk_examples/src/core_features/signCallback/useEncryptionSignCallback.ts

```diff
@@ -0,0 +1,30 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
+import { naclSeal } from '@polkadot/util-crypto'
+
+export function useEncryptionSignCallback({
```

Comment:
What's an `encryptionSignCallback`? Do you mean `encryptionCallback`?

Reply:
I have updated it :)